modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/saml/AbstractSAMLSSOTestCase.java

```diff
@@ -66,6 +67,8 @@
     private static final String NAMEID_FORMAT = "urn:oasis:names:tc:SAML:1.1:nameid-format:emailAddress";
     private static final String LOGIN_URL = "/carbon/admin/login.jsp";
 
+    protected static final String SAML_ECP_SSO_URL = "https://localhost:9853/samlecp";
+    protected static final String SAML_ECP_ACS_URL = "https://localhost/Shibboleth.sso/SAML2/ECP";
```

Comment:
Any reason for using "Shibboleth' in the ACS url?

Reply:
> Any reason for using "Shibboleth' in the ACS url?

Changed the ECP app ACS URL